docs/_templates/class.rst

```diff
@@ -63,6 +63,15 @@
    a pre-selected object (for example, people) detected in each specified zone. The order of counts
    follows the order of ``zones``.
 
+.. |btm_midpoint| replace:: ``btm_midpoint`` (:obj:`list`): (x ,y) coordinates of the bottom middle point of detected
+   boxes. The order of btm_mipoint follows the order of `bboxes`.
+
+ .. |all| replace:: ``all`` (:obj:`Any`): Uses all inputs produced in previous nodes. For more information on inputs,
+    see section under node data mapping.
```

Comment:
Not sure if we want a unified definition for `all`. See: https://github.com/aimakerspace/PeekingDuck/blob/313e530458ec6e3f1203518d0aab88d45b1277a0/peekingduck/pipeline/nodes/output/csv_writer.py#L33

as the param to edit is different between nodes, I suggest for `all` to be written for every node, instead of a generic one

Reply:
Ok. removed all from defaults